### PR TITLE
Input validation user sign in 1

### PR DESCRIPTION
**Pull request by 44jovi**
- User input validation on user sign-in (sessions/new)
- Integration test written and passing
- Password validation to come when the same is updated in UsersController

### CODE REVIEW

File: controllers/sessions.js [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
     res.render("sessions/new", {});
   },
 
   Create: (req, res) => {
-    console.log("trying to log in");
+    const errors = validationResult(req);
+
+    if (!errors.isEmpty()) {
+      return res.status(422).render("sessions/new", {
+        message: JSON.stringify(errors.array()[0].msg).replaceAll('"', ""),
```
Comment (ddrmv) [inline controllers/sessions.js]: Can be changed to `message: errors.array()[0].msg,`, as per comment on previous PR.

File: controllers/users.js [hunk 2]
```diff
@@ -39,6 +40,14 @@ const UsersController = {
   },
 
   Create: (req, res) => {
+    const errors = validationResult(req);
+
+    if (!errors.isEmpty()) {
+      return res.status(422).render("users/new", {
+        message: JSON.stringify(errors.array()[0].msg).replaceAll('"', ""),
```
Comment (ddrmv) [inline controllers/users.js]: Same here `message: errors.array()[0].msg,`

**Review by ddrmv — APPROVED** [review]
Same comment for the error string line, everything else seems good to me.

**Comment by 44jovi** [comment]
[User input validation - views/sessions/new.hbs](https://trello.com/c/OYr5Weeu/85-user-input-validation-views-sessions-newhbs)

File: controllers/sessions.js (hunk 1 above)
Comment (44jovi) [reply controllers/sessions.js]: Resolved!

**Comment by G-Redos** [comment]
No merge conflicts.

**Pull request merged**